lib/jsdom/browser/BrowsingContext.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+const idlUtils = require("../living/generated/utils");
+
+// https://html.spec.whatwg.org/#windows
+class BrowsingContext {
+  // readonly windowProxy: WindowProxy
+  // readonly sessionHistory: SessionHistory
+
+  // create a new browsing context
+  constructor() {
+    this.windowProxy = new WindowProxy();
```

Comment:
If we don't implement WindowProxy for now, we could just say `this.windowProxy = window` with a comment about how it's an interim solution, but the intent is that the public API presented by BrowsingContext can be reused later even when we implement a proper WindowProxy.